frontend/server/src/Controllers/Identity.php

```diff
@@ -408,6 +408,60 @@ private static function saveIdentityGroupInsideTransaction(
         );
     }
 
+    /**
+     * Save object Identities in DB, and add user into team group.
+     * This function is expected to be called inside a transaction.
+     */
+    private static function saveIdentityTeamInsideTransaction(
+        \OmegaUp\DAO\VO\Identities $identity,
+        \OmegaUp\DAO\VO\TeamGroups $teamGroup
+    ): void {
+        if (is_null($identity->username)) {
+            throw new \OmegaUp\Exceptions\NotFoundException(
+                'userNotExist'
+            );
+        }
+        if (is_null($identity->country_id) && !is_null($identity->state_id)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterInvalidStateNeedsToBelongToCountry',
+                $identity->username
+            );
+        } elseif (
```

Comment:
```suggestion
        }
        if (
```

aquí no es necesario el `else` porque el bloque de arriba va a interrumpir el flujo.